lib/beaker/hypervisor/docker.rb

```diff
@@ -70,6 +70,17 @@ def provision
         container_opts = {
           'Image' => image_name,
           'Hostname' => host.name,
+          'name' => host.name,
```

Comment:
I think this introduced a regression. The SUT containers are now named the same as their hostname, whereas before they had randomly generated names. When running two instances of the same nodeset simultaneously, or two separate nodesets with overlapping hostnames, beaker fails with the following error message:
```
An error occurred while loading ./spec/acceptance/class_spec.rb.

Failure/Error: require 'beaker-rspec'

Docker::Error::ConflictError:

  Conflict. The container name "/centos-6-x64" is already in use by container "5d843a8281173c6d3048432301fffc625301816eeba9c945822525a8a195001f". You have to remove (or rename) that container to be able to reuse that name.
```